src/apps/backend/modules/comment/rest-api/comment-serializer.ts

```diff
@@ -0,0 +1,18 @@
+import { Comment } from '../types';
+
+export const serializeCommentAsJSON = (comment: Comment): unknown => ({
+  id: comment.id,
+  task: comment.task,
+  account:
+    typeof comment.account === 'string'
+      ? { id: comment.account }
```

Comment:
if an account is a string then just do comment. account no need for an id field inside the object.